Documentation/CRDs/Cluster/external-cluster/topology-for-external-mode.md

```diff
@@ -0,0 +1,199 @@
+# Topology provisioning for external mode
+
+## Background
+For Block storage (RWO), Enable Topology awareness for PV provisioning. Preferably, picking the replica 1 Storage Class for this AZ or creating a volume in a relevant AZ using a single storage class. It will help in providing Isolation to application/pods in AZ
```

Comment:
I still don't understand this paragraph. Can you try to clarify it, and also add a sentence why it is related to external clusters?

Reply:
removed